### DISCUSSION TITLE
Taro小插件：编译完微信小程序后自动打开微信开发者工具

### DISCUSSION THREAD

**Discussion by 15380831711** [NervJS/taro#10831] [Q & A]
不知道有没有重复造轮子，原理很简单，就是调用了[微信开发者的命令行工具](https://developers.weixin.qq.com/miniprogram/dev/devtools/cli.html)
https://www.npmjs.com/package/taro-plugin-weapp-cli
感谢taro提供的插件支持！

**Comment by Chen-jj** [comment]
🐂哇

**Reply by 15380831711** [reply]
哈哈！第一次用taro插件功能，就是克隆了这个插件：https://github.com/toknT/taro-plugin-sync-in-wsl
